From 7593f30db42c25c26a211ac58f4f840fc4860d29 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <b02247@freescale.com>
Date: Thu, 29 Aug 2013 16:37:26 +0800
Subject: [PATCH 468/486] ENGR00276706 can't recognize the usb audio in ARD
 when HDMI connected

In the ARD, there is SPDIF, ESAI, HDMI, USB-AUDIO. the maximum card
num must be 4.

Signed-off-by: Shengjiu Wang <b02247@freescale.com>
---
 alsa/audio_hardware.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/alsa/audio_hardware.h b/alsa/audio_hardware.h
index 0d8c9c6..19f11fd 100644
--- a/alsa/audio_hardware.h
+++ b/alsa/audio_hardware.h
@@ -81,8 +81,8 @@ struct audio_card{
     int  in_format;
 };
 
-#define MAX_AUDIO_CARD_NUM  3
-#define MAX_AUDIO_CARD_SCAN 3
+#define MAX_AUDIO_CARD_NUM  4
+#define MAX_AUDIO_CARD_SCAN 4
 
 #define MAX_SUP_CHANNEL_NUM  20
 #define MAX_SUP_RATE_NUM     20
-- 
1.8.0

